monetdbe/_cffi/internal.py

```diff
@@ -296,3 +285,14 @@ def get_columns(self, table: str, schema: str = 'sys') -> Iterator[Tuple[str, in
             name = ffi.string(names_p[0][i]).decode()
             type_ = types_p[0][i]
             yield name, type_
+
+
+from monetdbe._cffi.branch import newer_then_jul2021
+if not newer_then_jul2021:
+    def prepare(self, query: str) -> monetdbe_statement:
+        self._switch()
+        stmt = ffi.new("monetdbe_statement **")
+        check_error(lib.monetdbe_prepare(self._monetdbe_database, str(query).encode(), stmt))
+        return stmt[0]
+
+    setattr(Internal, 'prepare', prepare)
```

Comment:
ah yes, see :) 

in case you want to silent mypy it is also okay to add `# type: ignore`, but then you need to have a good reason.

Reply:
indeed

Reply:
I'll keep that in mind